src/bcf/mod.rs

```diff
@@ -92,10 +99,17 @@ impl Reader {
         }
     }
 
-    /// Return an iterator over all records of the VCF/BCF file.
-    pub fn records(&mut self) -> Records {
+    fn records(&mut self) -> Records<Self> {
         Records { reader: self }
     }
+
+    fn header(&self) -> &HeaderView {
+        &self.header
+    }
+
+    fn empty_record(&self) -> Record {
```

Comment:
You should be able to move the implementation of empty_record into the Trait, by using the Read::header() method, right?

Reply:
OK

Reply:
Arg, then I need to expose the `Rc<HeaderView>` member. Is this OK?